tsfresh/utilities/dataframe_functions.py

```diff
@@ -322,4 +326,26 @@ def normalize_input_to_internal_representation(df_or_dict, column_id, column_sor
         if kind_to_df_map[kind][column_value].isnull().any():
             raise ValueError("You have NaN values in your value column.")
 
+    # Roll the data frames if requested
+    if rolling:
```

Comment:
yes. For the time being we can just add a check or throw an error if the time stamps are not sampled uniformly.